cdc/entry/mounter.go

```diff
@@ -386,53 +388,79 @@ func (m *mounterImpl) mountRowKVEntry(tableInfo *TableInfo, row *rowKVEntry) (*m
 			whereHandle := true
 			col.WhereHandle = &whereHandle
 		}
-		values[colName] = col
+		cols[colName] = col
+	}
+	for _, col := range tableInfo.Columns {
```

Comment:
If the datums are old value (such as from a delete event), does it need to fill in the default value for nonexistent fields?